File: features/operational/contact_us/src/main/java/com/tokopedia/contactus/inboxticket2/domain/StepTwoResponse.java
package com.tokopedia.contactus.inboxticket2.domain;

import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

public class StepTwoResponse{

	@SerializedName("comment")
	private Comment comment;

	@SerializedName("is_success")
	private int isSuccess;

	public void setComment(Comment comment){
		this.comment = comment;
	}

	public Comment getComment(){
		return comment;
	}

	public void setIsSuccess(int isSuccess){
		this.isSuccess = isSuccess;
	}

	public int getIsSuccess(){
		return isSuccess;
	}
}

File: features/operational/contact_us/src/main/java/com/tokopedia/contactus/inboxticket2/domain/StepTwoResponse.kt
package com.tokopedia.contactus.inboxticket2.domain

import com.google.gson.annotations.SerializedName

data class StepTwoResponse(
        @SerializedName("ticket_reply_attach")
        val ticketReplyAttach: TicketReplyAttach?
) {
    data class TicketReplyAttach(
            @SerializedName("data")
            val ticketReplyAttachData: TicketReplyAttachData?
    ) {
        data class TicketReplyAttachData(
                @SerializedName("is_success")
                val isSuccess: Int = 0,
                @SerializedName("status")
                val status: String? = ""
        )
    }
}